app/controllers/api/current/exotel_call_sessions_controller.rb

```diff
@@ -25,11 +47,16 @@ def valid_patient_phone_number?
     parse_patient_phone_number.scan(/\D/).empty?
   end
 
-  def respond_in_plain_text(status)
-    head status, content_type: 'text/plain'
+  def respond_in_plain_text(status, text = '')
```

Comment:
Do we need this method anymore? Can we inline it?

Reply:
I think this method is nice because it sort of weakly encapsulates the whole ExotelController to ensure that it is always in 'text/plain'.